sdk/servicebus/azure-servicebus/azure/servicebus/__init__.py

```diff
@@ -34,5 +34,6 @@
     'ServiceBusSender',
     'ServiceBusSharedKeyCredential',
     'TransportType',
-    'AutoLockRenew'
+    'AutoLockRenew',
+    'AutoComplete'
```

Comment:
I see it being quite likely that a customer would use these together.
We should have samples/tests for that and see if there's any obvious ways to optimize for that experience.

Reply:
Honestly this is the sad part about not being "push mode only" since then we could just have them both be flags; (adam actually had an interesting proposal along those lines as a modification to my V1 proposal but more tightly scoping it to the generator).

In leiu of that, I'm hard pressed to think of solutions to combining that I really "like"; especially since AutoLockRenew also functions on sessions.  perhaps something like "with message.automate(lock_renew=True, complete=True) is the first thing that comes to mind off the cuff but it seems... forced.  I'll need to give this more thought, for now I'll add a test and add this as an item to our list for next preview?